docs/design.md

```diff
@@ -0,0 +1,92 @@
+# Placement Policy Scheduler Plugins
```

Comment:
I think just the link to the design doc (https://github.com/Azure/placement-policy-scheduler-plugins/pull/46/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R15) is sufficient. If there are updates or addition to design, it can be updated in the living doc and we don't need to mirror those changes in this local copy too.